linera-sdk/src/service/test_runtime.rs

```diff
@@ -371,7 +371,7 @@ where
         self
     }
 
-    /// Configures the `Blob` returned when fetching from the `BlobId` during the test.
+    /// Configures the blob returned when fetching from the given hash during the test.
```

Comment:
Optional nit: I usually keep the backticks if the word matches the name of a parameter. Not sure if it makes sense here

```suggestion
    /// Configures the `blob` returned when fetching from the given `hash` during the test.
```